Formula/ttf2eot.rb

```diff
@@ -4,8 +4,7 @@ class Ttf2eot < Formula
   url "https://github.com/wget/ttf2eot/archive/v0.0.3.tar.gz"
   sha256 "f363c4f2841b6d0b0545b30462e3c202c687d002da3d5dec7e2b827a032a3a65"
   # License: Derived from WebKit, so BSD/LGPL 2/LGPL 2.1.
-  # license ["LGPL-2.0", "BSD-*"] - pending https://github.com/Homebrew/brew/pull/7953
-  license "LGPL-2.0"
+  license any_of: ["LGPL-2.0", "BSD-*"]
```

Comment:
```suggestion
  license any_of: ["LGPL-2.0-only", "BSD-2-Clause"]
```

Based on: https://webkit.org/licensing-webkit/. Didn't see any indication of `-only` or `-or-later` so went with `-only`. Maybe another maintainer can find some more info?

If we're not sure about that, we should probably:

```suggestion
  license :cannot_represent
```

Reply:
I do see at the end of that page this note:

> WebKit open source web browser engine.
> Copyright (C) 2015 Apple, Inc.
>
> This library is free software; you can redistribute it and/or
> modify it under the terms of the GNU Library General Public
> License as published by the Free Software Foundation; either
> version 2 of the License, or (at your option) any later version.

Doesn't that make it clear the `-or-later` versioning is intended?

Reply:
No, that's part of the standard `-or-later` and `-only` license.

See the SPDX entries for [`LGPL-2.0-only`](https://spdx.org/licenses/LGPL-2.0-only.html#licenseText) and [`LGPL-2.0-or-later`](https://spdx.org/licenses/LGPL-2.0-or-later.html#licenseText). They both contain the above lines in the "How to Apply These Terms to Your New Libraries" section.

We really need to see the source files to see which header is used. That will be able to tell us whether `-only` or `-or-later` is more approriate.

Reply:
Ah, I see. Well, that's confusing 🙃 I'll look into WebKit's source files, then.

Reply:
Ok, I cloned the repo (from [this mirror](https://github.com/WebKit/webkit)) and ran a search in the source files. It seems pretty clear that the `-or-later` variant is the intended one:

```console
$ rg -l 'the Free Software Foundation; either' | wc -l
2982

$ rg -l 'the Free Software Foundation; version' | wc -l
0
```

See also [this commit](https://github.com/WebKit/webkit/commit/d8747ba99c068e844114611dcff182bfd014efa2) for an example of the typical license headers they use.

Reply:
(That said, it seems that in this case we'd need to use `all_of` rather than `any_of`, since each file is only licensed under one of the two licenses; WDYT?)